packages/forms-lt-validators/package.json

```diff
@@ -0,0 +1,49 @@
+{
+    "name": "@reactway/forms-lt-validators",
+    "version": "1.0.0-alpha.2",
+    "description": "Default validators to use with ReactWay forms",
+    "main": "dist/index.js",
+    "module": "dist/index.es.js",
+    "scripts": {
+        "build": "tsc -p .",
+        "build:prod": "rollup -c && tsc -p . --emitDeclarationOnly --declarationMap false",
+        "watch": "tsc -p . -w",
+        "test": "eslint \"src/**/*.ts*\" --max-warnings 0"
+    },
+    "keywords": [
+        "react",
+        "forms",
+        "validation",
+        "ts"
+    ],
+    "files": [
+        "dist",
+        "docs",
+        "*.js",
+        "!*.config.js"
+    ],
+    "types": "dist/index.d.ts",
+    "author": "Reactway <dev@reactway.com> (https://github.com/reactway)",
+    "license": "MIT",
+    "devDependencies": {
+        "@reactway-tools/rollup": "0.0.0",
+        "@reactway/eslint-config": "1.0.1",
+        "@types/iban": "0.0.30",
+        "@types/validator": "13.1.0",
+        "eslint": "6.8.0",
+        "prettier": "1.19.1",
+        "typescript": "3.8.3",
+        "rollup": "1.32.0",
+        "jest": "24.9.0",
+        "ts-jest": "24.3.0"
+    },
+    "dependencies": {
+        "@reactway/forms": "1.0.0-alpha.2",
+        "@reactway/forms-core": "1.0.0-alpha.2",
+        "@reactway/forms-validators": "1.0.0-alpha.2",
```

Comment:
I don't see why you should require validators dependency. It could work as a standalone.

Reply:
It is due to importing base validator props from constants in `forms-validators` package. I could redefine them in `forms-validators-lt`, since they are base props and should not change.